### DISCUSSION TITLE
WebStorm warns about unused default export

### DISCUSSION THREAD

**Discussion by jeremy0519** [nuxt/nuxt#29703] [Questions]
I recently migrated from VSCode to Webstorm. Now WebStorm warns about unused default export almost anywhere. For example, in a simple `nuxt.config.ts` file, it warns that `defineNuxtConfig` is unused. Any idea how to fix that? I don't quite like the solution of disabling `JSUnusedGlobalSymbols` in every file.
<img width="1506" alt="image" src="https://github.com/user-attachments/assets/4d02a09b-285e-4431-9805-0c36421be4f4">

**Comment by jeremy0519** [comment]
I'm using latest WebStorm 2024.3 and nuxt. In Webstorm setttings and typescript language server is set to the typescript package in node_modules instead of bundled. Also tried clean cache.

**Reply by TheAlexLichter** [reply]
This is something that the WebStorm team could improve. They could ensure that the nuxt.config.ts / .js wouldn't throw that error.

I suppose it'd be the best to raise that in their bug tracker.

**Reply by jeremy0519** [reply]
Thanks!
